content/source/docs/cloud/registry/using.html.md

```diff
@@ -105,35 +90,54 @@ module "vpc" {
 }
 ```
 
-Configurations that reference modules via the generic hostname can be used without modification on any Terraform Enterprise instance, which is not possible when using hardcoded hostnames.
+~> **Important:** The generic hostname only works within a Terraform Enterprise instance.
+
+To test configurations on a developer workstation without the remote backend configured, you must replace the generic hostname with a literal hostname in all module sources and then change them back before committing to VCS. We are working on making this workflow smoother, but we currently only recommend `localterraform.com` for large organizations that use multiple Terraform Enterprise instances.
+
+### Module Availability
 
-~> **Important:** `localterraform.com` only works within a Terraform Enterprise instance — when run outside of Terraform Enterprise, Terraform can only use private modules with a literal hostname. To test configurations on a developer workstation without the remote backend configured, you must replace the generic hostname with a literal hostname in any module sources, then change them back before committing to VCS. We are working on ways to make this smoother in the future; in the meantime, we only recommend `localterraform.com` for large organizations that use multiple Terraform Enterprise instances.
+A workspace can only use private modules from its own organization's registry. When using modules from multiple organizations in the same configuration, we recommend:
+
+- **Terraform Cloud:** [Add modules to the registry](./publish.html#sharing-modules-across-organizations) for each organization that requires access.  
+
+- **Terraform Enterprise:** Check your site's [module sharing](../../enterprise/admin/module-sharing.html) configuration. Note that in Terraform Enterprise v202012-1 or higher, workspaces can also use private modules from organizations that are sharing modules with the workspace's organization.
 
 ## Running Configurations with Private Modules
 
-### In Terraform Cloud
+### Version Requirements
+
+Terraform 0.11 or later is required to:
+
+- Use private modules in Terraform Cloud workspaces with no extra setup.
```

Comment:
@hashitop What would users do if they have an earlier version? Is there another section in the docs that describes the setup they need to do?